common/focuses/00_lifestyle_focuses.txt

```diff
@@ -26,20 +26,6 @@ diplomacy_foreign_affairs_focus = {
 			}
```

Comment:
Still spends in different trees without finishing them. It's due to vanilla perks and focuses using different logic:
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132772763-e3d7f12f-1976-4e74-97b0-a74015123bfa.png)

</details>
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132772808-b1057ba7-721f-4bcf-874a-073f5598fec7.png)

</details>
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132771954-817d5e39-2d36-4360-911e-cf7362787ad9.png)
![image](https://user-images.githubusercontent.com/46384992/132771982-bdbcd563-2717-4645-9572-6f491d55ccc9.png)

</details>

Reply:
That's fine
Otherwise it will be impossible to give characters exact amount of magical perks that match the minimum required for their current level
So, with vanilla logic, we can't have 1, 2 and 4 level magicians in the game, since they all would require a fully finished tree, which is only applicable to a level 3/5.